utilities/cassandra/cassandra_functional_test.cc

```diff
@@ -318,6 +319,85 @@ TEST_F(CassandraFunctionalTest, CompactionShouldRemoveTombstoneFromPut) {
   ASSERT_FALSE(std::get<0>(store.Get("k1")));
 }
 
+#ifndef ROCKSDB_LITE
+TEST_F(CassandraFunctionalTest, LoadMergeOperator) {
+  ConfigOptions config_options;
+  std::shared_ptr<MergeOperator> mo;
+  config_options.registry->AddLibrary("cassandra", RegisterCassandraObjects,
```

Comment:
Add a check when config_options.registry->AddLibrary is not called and then call CreateFromString.